com.unity.visualeffectgraph/Editor/Utilities/pCache/BakeTool/PointCacheBakeTool.cs

```diff
@@ -1,10 +1,12 @@
+using System.Collections;
+using System.Collections.Generic;
```

Comment:
I guess those includes were unused ones that were removed by your first PR and have been therefore rolled back by your second?